module/applications/global/lp-history.mjs

```diff
@@ -0,0 +1,95 @@
+
+import LpTrackingData from "../../data/advancement/lp-tracking.mjs";
+
+/**
+ * 
+ * @param {object} inputs actor data
+ * @returns 
+ */
+export async function getLegendPointHistoryData( inputs ) {
+  return new Promise( ( resolve ) => {
+    let history = new LegendPointHistoryEarnedPrompt( inputs, resolve );
+    history.render( true );
+  } )
+}
+
+/**
+ * The application responsible for handling Legend Point History of Earned Points
+ *
+ * @augments {FormApplication}
+ *
+ * @param {LegendPointHistoryEarnedPrompt} legendpointHistoryEarned         The data model which is the
+ *      target data structure to be updated by the form.
+ * @param {FormApplicationOptions} [options={}]     Additional options which
+ *      modify the rendering of the sheet.
+ */
+export default class LegendPointHistoryEarnedPrompt extends FormApplication {
+  constructor( legendpointHistoryEarned , options = {} ) {
+    legendpointHistoryEarned ??= new LpTrackingData()
+    super( legendpointHistoryEarned );
+  }
+
+  /**
+   * Wait for dialog to be resolved.
+   * @param {object} [data]           Initial data to pass to the constructor.
+   * @param {object} [options]        Options to pass to the constructor.
+   */
+  static async waitPrompt( data, options = {} ) {
+    data ??= new LpTrackingData();
```

Comment:
What does this function do? It's named analogous to some other prompts which open it. Here it Only assigns some value to an unused variable/parameter.

Is this supposed to be the `getLegendPointHistoryData` function from above?